src/components/02-components/data-table/data-table-experiences.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {
+  Typography,
+  useTheme,
+} from '@mui/material';
+import OpenInBrowserIcon from '@mui/icons-material/OpenInBrowser';
+import EmailIcon from '@mui/icons-material/Email';
+import {
+  DataGrid,
+  GridColDef,
+  GridToolbarContainer,
+  GridToolbarExport,
+  GridToolbarColumnsButton,
+  GridToolbarFilterButton,
+  GridToolbarDensitySelector,
+  GridRowId,
+  GridActionsCellItem,
+} from '@mui/x-data-grid';
+
+export type Row = {
+  id: GridRowId;
+  col1: string;
+  col2: string;
+  col3: string;
+  col4: string;
+  col5: string;
+  col6: string;
+};
+
+interface DataTableProps {
+  initialRows: Row[];
+}
+
+export default function DataTableExperiences({ initialRows }: DataTableProps) {
+  const theme = useTheme();
+
+  function emailUser(col3: string) {
+    return () => {
+      window.location.href = 'mailto:' + col3;
+    };
+  }
+
+  function viewExperience(id: GridRowId) {
+    return () => {
+      window.location.href = '/organization/experiences/' + id;
```

Comment:
We need to avoid `window.location.href` when navigating. Is there other way to use `Button Link` component instead of `GridActionsCellItem` for link actions? 

Also notice the url shouldn't be hardcoded here, it needs to be passed down from the FE app instead